bin/release.sh

```diff
@@ -26,11 +26,16 @@ if [[ $continue_prompt != 'y' ]]; then
     exit 0
 fi
 
-NPM_USER=$(npm whoami)
-if ! [ "digital-marketplace" == "$NPM_USER" ]; then
+npm run test
```

Comment:
We probably want the release to not proceed if the tests fail 🤔

Reply:
I tested this and the whole script blows up if npm tests fails and outputs the various failed tests.

Reply:
Great :D